libraries/Matter/examples/MatterFan/MatterFan.ino

```diff
@@ -49,7 +49,11 @@ void fanDCMotorDrive(bool fanState, uint8_t speedPercent) {
   // drive the Fan DC motor
   if (fanState == false) {
     // turn off the Fan
-    digitalWrite(dcMotorPin, LOW);
+ #ifndef RGB_BUILTIN
+    // after analogWrite(), it is necessary to set the GPIO to digital mode first
+    pinMode(ledPin, OUTPUT);
```

Comment:
I think you meant `dcMotorPin` instead of `ledPin`

Reply:
Thanks!